packages/faceted-search/src/components/AddFacetPopover/AddFacetPopover.component.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { getTheme } from '@talend/react-components/lib/theme';
+import RichLayout from '@talend/react-components/lib/RichTooltip/RichLayout';
+import Action from '@talend/react-components/lib/Actions/Action';
+import FilterBar from '@talend/react-components/lib/FilterBar';
+
+import cssModule from './AddFacetPopover.scss';
+import { badgesFacetedPropTypes, badgeFacetedPropTypes } from '../facetedSearch.propTypes';
+
+const theme = getTheme(cssModule);
+
+const AddFacetRow = ({ badgeDefinition, id, label, onClick }) => {
+	const onClickRow = event => {
+		onClick(event, badgeDefinition);
+	};
+	return (
+		<Action
+			className={theme('tc-add-facet-popover-row')}
+			id={`${id}-row-button-${label}`}
+			onClick={onClickRow}
+			label={label}
+			link
+			role="button"
+		/>
+	);
+};
+
+AddFacetRow.propTypes = {
+	id: PropTypes.string.isRequired,
+	badgeDefinition: badgeFacetedPropTypes.isRequired,
+	label: PropTypes.string.isRequired,
+	onClick: PropTypes.func.isRequired,
+};
+
+const filterByAttribute = attribute => badgeDefinition =>
```

Comment:
I think you need to trim/downcase filter attribute: If a search `Name`, it didn't filter `Name` badge. On the other side, `name` find `Name`. Same for `    name` which doesn't works.